views/student.ejs

```diff
@@ -0,0 +1,19 @@
+<%- include("partials/head.ejs") %>
+<%- include("partials/header.ejs") %>
+<h2><%= title %></h2>
+<form method="POST" action="<%= action %>">
+  <label for="schoolName">School Name:</label>
+  <input type="text" id="schoolName" name="SchoolName" value="<%= student ? student.SchoolName : '' %>"><br>
+
+  <label for="grade">Grade:</label>
+  <input type="text" id="grade" name="Grade" value="<%= student ? student.Grade : '' %>"><br>
+
+  <label for="subject">Subject:</label>
+  <input type="text" id="subject" name="Subject" value="<%= student ? student.Subject : '' %>"><br>
```

Comment:
In the future, I would suggest making something like this a dropdown instead of a text entry. If I enter a subject that isn't on the list, it throws an error, and doesn't say what are acceptable values anywhere. I'll approve it, but keep this in mind for the future.

Reply:
Sure, I will add dropdown instead of a text entry. Thank you